Dockerfile

```diff
@@ -6,4 +6,5 @@ RUN apk add --no-cache git make && \
 
 FROM alpine:3.13
 COPY --from=build /s5cmd/s5cmd .
-ENTRYPOINT ["./s5cmd"]
+WORKDIR /aws
```

Comment:
What do think about making it a vendor agnostic name like `src` or something? 

Also it would be good to document the working directory in the Docker section of the README with an example.

Reply:
My reasoning was to unify the usage since s5cmd also supports some AWS_* env variables. It would be easier to jump from one image to another.